tdms/CMakeLists.txt

```
@@ -61,6 +64,25 @@ else()
 endif()
 
 
+# spdlog ----------------------------------------------------------------------
+find_package(spdlog NO_CMAKE_PACKAGE_REGISTRY QUIET)
+if(NOT spdlog_FOUND)
+    include(FetchContent)
+    FetchContent_Declare(spdlog
+            GIT_REPOSITORY https://github.com/gabime/spdlog.git
+            GIT_TAG v1.10.0
+            )
+    FetchContent_MakeAvailable(spdlog)
+endif()
+
+# Set the active logging level
+if("${CMAKE_BUILD_TYPE}" STREQUAL "Debug" OR "${CMAKE_BUILD_TYPE}" STREQUAL "${CMAKE_BUILD_TYPE}" STREQUAL "RelWithDebInfo")
```

sure this is right @giordano ?! I would have thought

```suggestion
if("${CMAKE_BUILD_TYPE}" STREQUAL "Debug" OR "${CMAKE_BUILD_TYPE}" STREQUAL "RelWithDebInfo")
```

---

Right.  I said that the original suggestion got badly garbled.  Actually running the tests would probably have caught this early on though.